### PR TITLE
symplified internal jsonColorRgb handling

### PR DESCRIPTION
**Pull request by moinmoin-sh**
I symplified the internal `jsonColorRgb `handling. A common function converts the json array to a `lv_color_t `which is used internally instead of the` r,g,b's`

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1960,9 +1934,9 @@ void jsonTileCommand(JsonVariant json)
 
     if (jsonColorPicker.containsKey("colorRgb"))
     {
-      r = (uint8_t)jsonColorPicker["colorRgb"]["r"].as<int>();
-      g = (uint8_t)jsonColorPicker["colorRgb"]["g"].as<int>();
-      b = (uint8_t)jsonColorPicker["colorRgb"]["b"].as<int>();
+      uint8_t r = (uint8_t)jsonColorPicker["colorRgb"]["r"].as<int>();
```
Comment (sumnerboy12) [inline src/main.cpp:1937]: Any reason you didn't do it for this as well?

**Review by sumnerboy12 — APPROVED** [review]
Nice tidy up

File: src/main.cpp (hunk 1 above)
Comment (moinmoin-sh) [reply src/main.cpp:1937]: yes, this is a 32-bit color and needs a different handling.

Comment (sumnerboy12) [reply src/main.cpp:1937]: Could still convert the JSON payload to `lv_color_t` and then convert to 32-bit color inside `tile.setColorPickerRgb()`?

Just means the internal API for sending colours around is consistent.

But up to you, I am happy either way.

Comment (moinmoin-sh) [reply src/main.cpp:1937]: that doesn't work. The standard color format for the screen is RGB565 (16-bit). The conversion to lv_color_t is always done to match the systems RBG resolution. So when r,g,b is converted to lv_color_t the lower bits are lost and a lossless conversion to RGB88 is not possible.

**Pull request merged**